data/map planets.txt

```diff
@@ -3386,8 +3447,12 @@ planet "New Greenland"
 	description `New Greenland was first settled about six hundred years ago, at the peak of the "Turf Wars" era where most planets were controlled by powerful corporations and their struggles often turned into outright warfare with privately owned fleets and armies. The people who came to New Greenland were looking for a world so marginal and undesirable that they could live here in peace without being caught in the crossfire.`
 	description `	The locals here grow food in greenhouses and specialize in making warm winter clothing, which is this planet's only export.`
 	spaceport `The spaceport is near the equator, but it is still cold enough that there is always some snow on the ground.`
-	spaceport `	You have never seen as many people in fur coats as there are here; what might be considered a luxury in other parts of the galaxy is a necessity here, where the cheapest clothing material is the pelts of indigenous animals.`
+	spaceport `	You have never seen as many people in fur coats as there are here; what might be considered a luxury elsewhere is a necessity here, where the cheapest clothing material is the pelts of indigenous animals.`
```

Comment:
~~unrelated change~~
And also, I'm not sure this is better; elsewhere could make it sound as if elsewhere on-planet.
```suggestion
	spaceport `	You have never seen as many people in fur coats as there are here; what might be considered a luxury in other parts of the galaxy is a necessity here, where the cheapest clothing material is the pelts of indigenous animals.`
```

Reply:
Without the change, the text ends up going too low. There's supposed to be at least a 10-pixel margin at the top and bottom of the box; having the extra text reduces the margin to 7 pixels at the bottom.

![image](https://github.com/user-attachments/assets/f84c2efb-fad5-42ef-9dee-90a1a8a97b7e)